docs/sources/enterprise/query-caching.md

```diff
@@ -21,20 +21,27 @@ The following cache backends are available: in-memory, Redis, and Memcached.
 - Reduced API costs.
 - Reduced likelihood that APIs will rate-limit or throttle requests.
 
-## Enable query caching
+## Enable and configure query caching
 
-> **Note:** The query caching feature is enabled by default. To start caching data source queries, visit the data source's configuration page.
+By default, data source queries are not cached. To enable query caching for a single data source:
+1. Visit the data source settings page.
+1. Open the Cache tab.
+1. Press the Enable button. 
+1. Once enabled, you can choose a custom TTL for that data source or keep the default TTL.
 
-To enable and configure query caching, please refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).
+> **Note:** If query caching is enabled and the Cache tab is not visible in a data source's settings, then query caching is not available for that data source.
 
-Once caching is enabled in the Grafana instance, query caching can be enabled per data source in the data source settings.
+To configure global settings for query caching, refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).
 
 ## Disable query caching
 
-To disable query caching for a single data source, visit the data source settings page.
+To disable query caching for a single data source: 
+1. Visit the data source settings page.
```

Comment:
Explain how to do this. 

1. On the side menu, click...
1. In the data source list, click the data source that you want to turn off caching for.
1. In the Cache tab, click **Disable**.

Reply:
Applied :)